package.json

```diff
@@ -21,7 +21,6 @@
     "build:watch": "lerna run --parallel watch",
     "pretest": "lerna run lint",
     "test": "npm run test:only",
-    "pretest:only": "npm run build",
```

Comment:
i usually run `npx mocha --recursive packages/enzyme-test-suite/test` directly, to avoid this (because i run the build watcher in another tab)

Reply:
Ah `npx`. I'm used to `yarn` where I it automatically uses the binary if the script can't be found. Thanks for the tip. Might be good to include in CONTRIBUTING.md.